empress/reconcile/median.py

```diff
@@ -93,36 +93,36 @@ def generate_scores(preorder_mapping_node_list, dtl_recon_graph, gene_root, norm
             # This will also populate the counts dictionary with the number of MPRs each event and mapping node is in
             count += count_mprs(mapping_node, dtl_recon_graph, counts)
 
-    # Initialize the scores dict. This dict contains the frequency score of each mapping node
-    scores = dict()
+    # This dict contains the frequency score of each mapping node
+    node_frequencies = dict()
     for mapping_node in preorder_mapping_node_list:
-        scores[mapping_node] = 0.0
+        node_frequencies[mapping_node] = 0.0
 
     # This entry is going to be thrown away, but it seems neater to just let calculateScoresOfChildren
     # add scores to an unused entry than to check to see if they are (None, None) in the first place.
-    scores[(None, None)] = 0.0
+    node_frequencies[(None, None)] = 0.0
 
     # The scored graph is like the DTLReconGraph, except instead of individual events being in a list, they are the
```

Comment:
Same as the comment above, I think we are not actually explicitly constructing a "scored graph" with mapping nodes as keys and events and their frequencies as values, so this comment can be misleading